### ISSUE TITLE
Delete the default font appears blank

### ISSUE THREAD

**Issue by bszhct** [ONLYOFFICE/DocumentServer#1494]
**Do you want to request a *feature* or report a *bug*?**
Bug?

**What is the current behavior?**
Delete the default font appears blank

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
I deleted the font files in the core-fonts directory, and then executed /usr/bin/documentserver-generate-allfonts.sh to repackage it, and there was a blank space in the font list on the front end

![image](https://user-images.githubusercontent.com/56250514/139665282-e9e7f54c-6c0c-49e9-88bf-6c9ffba8be2f.png)
![image](https://user-images.githubusercontent.com/56250514/139665927-de2cf54f-642c-4c3f-9f25-431447bae27e.png)

**What is the expected behavior?**
Deleting the default font will not appear blank and will no longer display the default font

**Did this work in previous versions of DocumentServer?**
Uncertain

**DocumentServer version:**
6.4.2

**Operating System:**
macOS Mojave 10.14.5

**Browser version:**
95.0.46xxx

**Comment by ShockwaveNN** [comment]
I think you should clean web cache or open browser in incognito mode, this will fix it

**Comment by bszhct** [comment]
I use these two methods to clean up the cache, but there are still blanks.

![image](https://user-images.githubusercontent.com/56250514/139667048-5c7f2c0d-e6a3-4519-afce-fac667242c9e.png)
![image](https://user-images.githubusercontent.com/56250514/139667060-4ea22514-eef5-4274-abfa-90b9c45b7d8f.png)

**Comment by bszhct** [comment]
Isn't it clearing the cache like me?

**Comment by ShockwaveNN** [comment]
As far as I'm aware - you're doing ok, not understand your language but the screenshot seems ok

I've rechecked on my system: 

1. Run default docker image with `docker run -itd -p 80:80 onlyoffice/documentserver`
2. Removed all default files via:
```
cd /var/www/onlyoffice/documentserver/core-fonts
rm -rf *
```
3. Run `documentserver-generate-allfonts.sh`
4. Open new doc in incognito:
![image](https://user-images.githubusercontent.com/668524/139669880-d3c150aa-d47b-4847-b25d-9c6643a49f8a.png)
![image](https://user-images.githubusercontent.com/668524/139669924-72265d5e-a9f5-498e-8b00-c4202faa7e09.png)

Everything look ok, several default system languages is still present and there are no empty lines

Maybe it's some caching on DMS side, judging by URL `/editor-template` it's not our solution
